example-app/SantokuApp/src/features/demo-map/pages/DemoMapPage.tsx

```diff
@@ -0,0 +1,267 @@
+    (values: RegionFormValues) => {
+      const latitude = Number(values.latitude);
+      const longitude = Number(values.longitude);
+      const latitudeDelta = Number(values.latitudeDelta);
+      const longitudeDelta = Number(values.longitudeDelta);
+      setRegion({latitude, longitude, latitudeDelta, longitudeDelta});
+      setDefaultMarker({coordinate: {latitude, longitude}});
+    },
+    [setRegion],
+  );
+  const {form} = useRegionForm({
+    initialValues: {
+      latitude: initialRegion.latitude.toString(),
+      longitude: initialRegion.longitude.toString(),
+      latitudeDelta: initialRegion.latitudeDelta.toString(),
+      longitudeDelta: initialRegion.longitudeDelta.toString(),
+    },
+    onSubmit,
+  });
+  const onMarkerFormSubmit = useCallback((values: MarkerFormValues) => {
+    // 緯度と経度が入力されていた場合のみマーカーを追加し、そうでない場合はマーカーをクリアする
+    if (values.latitude && values.longitude) {
+      const marker = {
+        coordinate: {
+          latitude: Number(values.latitude),
+          longitude: Number(values.longitude),
+        },
+        title: values.title,
+        description: values.description,
+        draggable: values.draggable,
+      };
+      setFormSubmittedMarker(marker);
+    } else {
+      setFormSubmittedMarker(undefined);
+    }
+  }, []);
+  const {form: markerForm, setMarkerFormDraggable} = useMarkerForm({onSubmit: onMarkerFormSubmit});
+
+  const keyboardType = Platform.select({
+    ios: 'numbers-and-punctuation',
+    android: 'number-pad',
+  } as const);
+
+  const [scrollEnabled, setScrollEnabled] = useState<boolean>(true);
+  const [zoomEnabled, setZoomEnabled] = useState<boolean>(true);
+  const [rotateEnabled, setRotateEnabled] = useState<boolean>(true);
+  const [pitchEnabled, setPitchEnabled] = useState<boolean>(true);
+
+  const mapViewRef = useRef<MapViewRef>(null);
+
+  useEffect(() => {
+    mapViewRef.current?.animateToRegion(region);
+  }, [region]);
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.flexContainer}>
+        <ScrollView>
+          <View style={styles.flexContainer}>
+            <SpecAndSourceCodeLink feature="map" />
+            <Text>
+              {`
+地図を表示します。オプションを変更し、表示や操作を確認できます。
+        `}
+            </Text>
+            <SectionBox title="表示領域の変更" description="表示領域を変更できます。">
+              {regionFormFields.map(({formKey, description}, index) => (
+                <FieldBox key={`region-field-${index}`} title={formKey} description={description}>
+                  <TextInput
+                    value={form.values[formKey]}
+                    onChangeText={form.handleChange(formKey)}
+                    errorMessage={form.errors[formKey]}
+                    keyboardType={keyboardType}
+                  />
+                </FieldBox>
+              ))}
+            </SectionBox>
+            <View style={styles.search}>
+              <Button title="更新する" onPress={form.submitForm} size="middle" />
+            </View>
```

Comment:
マーカーと違ってこっちは更新ボタンが SectionBox から出てました
```suggestion
              <View style={styles.search}>
                <Button title="更新する" onPress={form.submitForm} size="middle" />
              </View>
            </SectionBox>
```

Reply:
`SectionBox`内に移動しましたー
9974038f9de55a7a2b53835408c3982a0d98eb96